src/components/ChattingBubble/index.tsx

```diff
@@ -0,0 +1,58 @@
+import type { HTMLAttributes, ReactElement } from 'react'
+import styled from '@emotion/styled'
+
+export interface ChattingBubbleProps extends HTMLAttributes<HTMLDivElement> {
+  type: 'send' | 'receive'
+  children: string
+}
+
+interface StyledBubbleProps {
+  type: 'send' | 'receive'
+  isSend: boolean
+}
+
+export const ChattingBubble = ({
+  type,
+  children,
+  ...props
+}: ChattingBubbleProps): ReactElement => {
+  const newLineRegex = /\s/gi
+  const blankChatLength = children.length
+  const noBlankChatLength = children.replace(newLineRegex, '').length
+  const overChatLength = blankChatLength - noBlankChatLength
+  const isSend = type === 'send'
+
+  return (
+    <StyledBubble {...props} isSend={isSend} type={type}>
+      {noBlankChatLength <= 100
+        ? children
+        : children?.substring(0, 100 + overChatLength)}
+    </StyledBubble>
+  )
+}
+
+const StyledBubble = styled.p<StyledBubbleProps>`
+  display: inline-block;
+  word-break: break-all;
+  max-width: 332px;
+  max-height: 144px;
+  padding: 12px 16px;
+  color: ${({ theme, isSend }): string =>
+    isSend ? theme.colors.grayScale.white : theme.colors.grayScale.gray90};
+  ${({ theme, isSend }): string =>
+    isSend ? theme.fonts.body01M : theme.fonts.body01R}
+  background-color: ${({ theme, isSend }): string =>
+    isSend ? theme.colors.brand.primary : theme.colors.grayScale.white};
+  border-radius: ${({ isSend }): string =>
+    isSend ? '16px 0px 16px 16px' : '0px 16px 16px 16px'};
```

Comment:
수림님 코드 보면 스타일을 한번에 불러와서 적용했더라구요 ! 
훨씬 가독성이 좋아서 저도 수정하려고 하는데 참고해보시믄 좋을 것 같아용ㅎㅎ

```suggestion
 ${({ theme, isSend }): string => {
    if(isSend){
       return `
          color: ${theme.colors.grayScale.white};
          background-color: ${theme.colors.brand.primary};
          border-radius: '16px 0px 16px 16px';`
          ${theme.fonts.body01M}
    }
    
    return `
       color: ${theme.colors.grayScale.gray90};
       background-color: ${theme.colors.grayScale.white};
       border-radius: '0px 16px 16px 16px';
       ${theme.fonts.body01R}
    `
 }
```